src/components/Test.md

```diff
@@ -0,0 +1,10 @@
+```jsx
+<div>
+<form onSubmit={e=>{e.preventDefault();console.log("hello")}} >
+
+  <Test label="test" required options={[{label:"test", value:"adadad"},{label:"hello", value:"moto"}, {label:"test", value:"coucou"},{label:"hello", value:"tzaeaze"},{label:"test", value:"rezrzerzerz"},{label:"hello", value:"vsdazeazeaze"},]} onChange={e=>console.log(e)}/>
+  <button>ok</button>
```

Comment:
remove button ?

Reply:
button is required to test the required attribute

Reply:
👍 
You will rename Test or I can approve ?